roles/dnsmasq/templates/resolv.conf.j2

```diff
@@ -0,0 +1,8 @@
+; generated by ansible
+search {{ [ 'default.svc.' + dns_domain, 'svc.' + dns_domain, dns_domain ] | union(domain_search_list) | unique | join(' ') }}
+{% for host in groups[master_group_name] %}
+nameserver {{ hostvars[host]['ansible_default_ipv4']['address'] }}
+{% endfor %}
+{% for nameserver in nameservers_list | unique %}
+nameserver {{ nameserver }}
+{% endfor %}
```

Comment:
Would the approach taken within MI work?
https://github.com/CiscoCloud/microservices-infrastructure/blob/master/roles/dnsmasq/templates/resolv.conf.j2